static/js/shared/stat_types.ts

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+import { StatMetadata } from "../tools/shared_util";
```

Comment:
nit: since it's a type from our API response, should we keep that here (in stat_types), and import into shared_util?

Reply:
done